t/util/reindex

```diff
@@ -1,27 +1,65 @@
-#!/usr/bin/env lua
-
-if not arg[1] then
-   io.stderr:write("Usage: "..arg[0].." t/*.t\n")
-   os.exit(1)
-end
-
-for _, name in ipairs(arg) do
-   local i = 1
-   local fd = io.open(name, "r")
-   if fd then
-      local new = name.."~"
-      local out = io.open(new, "w")
-      for line in fd:lines() do
-         local test, n, desc = line:match("^(===%s*TEST%s*)(%d+)(.*)$")
-         if test then
-            out:write(test .. tostring(i) .. desc .. "\n")
-            i = i + 1
-         else
-            out:write(line .. "\n")
-         end
-      end
-      out:close()
-      fd:close()
-      os.execute("mv " .. new .. " " .. name)
-   end
-end
+#!/usr/bin/env perl
+
+#: reindexes .t files for Test::Base based test files
+#: Copyright (c) Yichun Zhang
```

Comment:
What is the license of this file? I don't think we should include it in this repo, since is has someone else's copyright. The other implementation of reindex worked just fine, AFAICT. The main difference is not adding multiple blank lines above each test, but that is harmless.

Reply:
https://github.com/openresty/openresty-devel-utils#copyright--license

`2-clause BSD license`.

can you accept this version? If not, I will delete it.

Reply:
I _personally_ don't have problem with this license (or any other open source licenses for that matter! :) ), but I am concerned about adding code to the company's repo with someone else's copyright notice. The main Kong/kong repo does have a CLA signing step for PRs; this repo currently doesn't but it may add it in the future, so having a third-party copyright in it may complicate things. Please remove the script from this PR.

Reply:
removed new version ^_^